packages/flutter/test/foundation/leak_tracking.dart

```diff
@@ -35,6 +35,9 @@ class WeakSet {
   }
 }
 
+/// If true, the leak tracker will collect debug information for leaks.
+bool collectDebugInformationForLeaks = false;
```

Comment:
Is this a debug only flag? Then it should be named something like "debugFoo". It should also be moved to debug.dart. 

And the documentation should be extended.

Reply:
Removed it for now as it turned out I did not use it for debugging.